tutorials/3d/introduction_to_3d.rst

```diff
@@ -16,19 +16,19 @@ In 3D, math is a little more complex than in 2D, so also checking the
 developers, not mathematicians or engineers) will help pave the way for you
 to develop 3D games efficiently.
 
-Spatial node
+Node3D
 ~~~~~~~~~~~~
```

Comment:
Title underline length must match the text (this is a reStructuredText requirement):

```suggestion
Node3D
~~~~~~
```